tools/dbus_ignorelist

```diff
@@ -882,6 +882,7 @@ org.freedesktop.systemd1.Mount.ManagedOOMSwap
 org.freedesktop.systemd1.Mount.MemoryAccounting
 org.freedesktop.systemd1.Mount.MemoryAvailable
 org.freedesktop.systemd1.Mount.MemoryCurrent
+org.freedesktop.systemd1.Mount.MemoryPeak
```

Comment:
This file is for dbus methods / properties added before v250. For new properties please add a version info in the "History" section of org.freedesktop.systemd1.xml instead

Reply:
Done (hopefully correctly).